README.md

```diff
@@ -125,9 +125,6 @@ Ctl prints out the command before running.
 ## Config
 Cluster level configs are cached. To update this cached data, run `ctl config fetch`. Sometimes you may need to delete the ctl config folder. This is normally at `$XDG_CONFIG_DIR/ctl` or `~/.config/ctl`.
 
-## Kron
```

Comment:
I scanned through the code again and I actually couldn't find base ctl commands that cover suspend/unsuspend for cron jobs. I think exec may be covered with the run command